pages/assertions/Array-assertions-made-easy--e.g.-SIZE,-element-finder.md

```diff
@@ -1,9 +1,4 @@
 
```

Comment:
Hi, Can you please rename the file to below:
```
Array-assertions-made-easy-element-finder.md
```

Thanks.

Overall looks good.